src/components/bannerLogo/bannerLogo.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+
+import { IBannerLogo } from "../../types/common/ComponentsProps";
+
+import commonStyles from "./bannerLogo.module.scss";
+
+const BannerLogo = ({ smallLogo }: IBannerLogo) => {
+	return (
+		<div className={commonStyles["banner-logo"]}>
+			<Image
+				src={smallLogo ? "/logo_small(light).webp" : "/logo.webp"}
+				alt="Logo"
+				width={715}
+				height={85}
+				priority
+				className={`${smallLogo ? commonStyles["banner-logo__small-logo-image"] : commonStyles["banner-logo__logo-image"]}`}
+			/>
+		</div>
```

Comment:
banner-logo -> bannerLogo
banner-logo__small-logo-image -> bannerLogo__smallLogoImage
banner-logo__logo-image -> bannerLogo__mediumLogoImage

className={cn(styles.bannerLogo__mediumLogoImage, {[styles.bannerLogo__smallLogoImage]: smallLogo})}

can you rename this class name with classnames library?